swift/ios-shoppe-demo/Views/ProductCollectionViewCell.swift

```diff
@@ -25,6 +25,9 @@ class ProductCollectionViewCell: UICollectionViewCell {
     var collectionView: StoreViewController?
 
     func setup() {
+        // MARK: - FullStory Unmasking Example
+        fsModifyPrivacy(setting: .unmask, of: self)
```

Comment:
For all the `fsModifyPrivacy(setting: .unmask, of: self)`s, see if the variadic function makes setting privacy more consistent.

Reply:
I'm not sure I understand.